tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -399,8 +400,28 @@ def multi_popen(server_configs, server_name="", enable_redirect_log=False):
-        def send_request(prompt: str, sampling_params: SamplingParams,
+        request_counter = itertools.count()
+
+        def _format_request_log(request_id: int, prompt: str,
+                                sampling_params: SamplingParams,
+                                streaming: bool) -> str:
+            prompt_hash = hashlib.sha256(
+                prompt.encode("utf-8", errors="replace")).hexdigest()[:12]
+            prompt_preview = re.sub(r"\s+", " ", prompt).strip()[:240]
+            try:
+                prompt_tokens = len(tokenizer.encode(prompt))
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)